### PR TITLE
Show an error/warning when the HPB is incompatible

### PR DESCRIPTION
**Pull request by nickvergessen**
- [x] Show an error in the HPB configuration section

Bildschirmfoto von 2021-11-03 15-27-32

- [x] Show an error/warning for users

Bildschirmfoto von 2021-11-03 15-57-38

### CODE REVIEW

File: lib/Controller/SignalingController.php [hunk 1]
```diff
@@ -216,6 +216,11 @@ public function getWelcomeMessage(int $serverId): DataResponse {
 				],
 			]);
 
+			$featureHeader = $response->getHeader('X-Spreed-Signaling-Features');
+			if (strpos($featureHeader, 'audio-video-permissions') === false) {
```
Comment (fancycode) [inline lib/Controller/SignalingController.php:220]: While this works for now, better would be to split the comma-separated string and check the individual items.

**Review by fancycode — APPROVED** [review]
See comment above.

File: lib/Controller/SignalingController.php (hunk 1 above)
Comment (nickvergessen) [reply lib/Controller/SignalingController.php:220]: it's `, ` (comma + space) seperated, right?
I will split by command then then trim to be save

**Comment by nickvergessen** [comment]
Added screenshots

**Review by danxuliu — APPROVED** [review]
Tested and works :+1: 

It would be nice if the message in the admin section mentions the minimum compatible version, but I think it is also fine not to mention it so the admin tries with the latest one ;-)

File: src/utils/signaling.js [hunk 2]
```diff
@@ -962,6 +962,16 @@ Signaling.Standalone.prototype.helloResponseReceived = function(data) {
 		}
 	}
 
+	if (!this.features['audio-video-permissions']) {
```
Comment (fancycode) [inline src/utils/signaling.js:965]: Better:
```
if (!this.hasFeature('audio-video-permissions')) {
```
https://github.com/nextcloud/spreed/blob/0e1b20723f6882a04a326e565a22abd17c3e25f1/src/utils/signaling.js#L161

Comment (nickvergessen) [reply src/utils/signaling.js:965]: ah nice

**Pull request merged**